pkg/build/api/validation/validation.go

```diff
@@ -49,12 +49,12 @@ func ValidateBuildConfig(config *buildapi.BuildConfig) fielderrors.ValidationErr
 	for i, trg := range config.Triggers {
 		allErrs = append(allErrs, validateTrigger(&trg).PrefixIndex(i).Prefix("triggers")...)
 		if trg.Type == buildapi.ImageChangeBuildTriggerType {
-			ictCount++
+			if ictCount++; ictCount > 1 {
+				allErrs = append(allErrs, fielderrors.NewFieldInvalid("triggers", config.Triggers, "only one ImageChangeTrigger is allowed"))
```

Comment:
s/ImageChangeTrigger/ImageChange trigger/ ?

From the API perspective, there's no "ImageChangeTrigger"